sw/tools/airframe_editor/airframe_editor.py

```diff
@@ -1,37 +1,204 @@
 #!/usr/bin/env python
+# coding: utf-8
 
-from __future__ import print_function
+r"""Airframe editor"""
 
-import pygtk
-import gtk
-pygtk.require('2.0')
+from __future__ import print_function
 
-from os import path
+from os.path import join
```

Comment:
Personally I prefer to import not only the `join` but rather `path` and use `path.join` as for me this makes it more clear in the code what is joined (a lot of packages/classes have join methods).
But this is my personal preference and not a strict rule or anything...

Reply:
I agree with @flixr's argument. On the other hand, the former gives the entire `path` object/structure, when we just need the `join` function of it. As a compromise, what about

```python
from os.path import join as pathjoin
```